src/java.desktop/share/classes/javax/swing/table/DefaultTableColumnModel.java

```diff
@@ -290,6 +290,9 @@ public int getColumnIndex(Object identifier) {
      * @param   columnIndex     the index of the column desired
      * @return  the <code>TableColumn</code> object for the column
      *                          at <code>columnIndex</code>
+     * @exception  ArrayIndexOutOfBoundsException if <code>columnIndex</code>
+     *             is out of range:
```

Comment:
As Joe Darcy pointed out, we use @throws these days.

Reply:
I was aware of that but this file uses @exception throughout, so to be consistent, I used that..But if it is recommended to use @throws even at the cost of being inconsistent within the file, I would use that.